src/main/webapp/css/styles.css

```diff
@@ -10276,3 +10276,24 @@ body {
 body > .container {
   flex-grow: 1;
 }
+
+#header-bg-img {
+  background-image: url(/../assets/img/food-photo.jpeg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center center;
+}
+
+.home-deal-img {
+  height: 270px;
+}
+
+.see-more-link {
+  display:inline-block;
+  bottom: 0;
+}
```

Comment:
I don't see any of these being used in the HTML or JavaScript? Am I missing something?